lib/vector2_extension.dart

```diff
@@ -0,0 +1,68 @@
+export 'package:vector_math/vector_math_64.dart' show Vector2;
+
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:vector_math/vector_math_64.dart';
+
+extension Vector2Extension on Vector2 {
+  /// Creates an [Offset] from the [Vector2]
+  Offset toOffset() => Offset(x, y);
+
+  /// Creates a [Size] from the [Vector2]
+  Size toSize() => Size(x, y);
+
+  /// Creates a [Point] from the [Vector2]
+  Point toPoint() => Point(x, y);
+
+  /// Creates a [Rect] starting from [x, y] and going the [Vector2]
+  Rect toRect(Vector2 to) => Rect.fromLTWH(x, y, to.x, to.y);
+
+  /// Creates a [Rect] starting in origo and going the [Vector2]
+  Rect toOrigoRect() => Rect.fromLTWH(0, 0, x, y);
+
+  /// Linearly interpolate towards another Vector2
+  void lerp(Vector2 to, double t) {
+    setFrom(this + (to - this) * t);
+  }
+
+  /// Rotates the [Vector2] with [angle] in radians
+  void rotate(double angle) {
```

Comment:
the vector2 lib doesn't have this???

Reply:
Not in a simple way like this that I could find at least, very strange.